src/app/child-dev-project/attendance/attendance.service.ts

```diff
@@ -254,19 +264,34 @@ export class AttendanceService {
     date: Date
   ): Promise<EventNote> {
     const instance = new EventNote();
-    const childIdPromises = activity.linkedGroups.map((groupId) =>
-      this.childrenService
-        .queryRelationsOf("school", groupId)
-        .then((relations) => relations.map((r) => r.childId))
+    const schoolParticipants = await this.loadParticipantsOfGroups(
+      activity.linkedGroups
     );
-    const schoolParticipants = await Promise.all(childIdPromises);
     instance.date = date;
     instance.subject = activity.title;
     instance.children = [
       ...new Set(activity.participants.concat(...schoolParticipants)), //  remove duplicates
     ];
+    instance.schools = activity.linkedGroups;
```

Comment:
Maybe it was not to best idea to call it `linkedGroups` for now we still use the school term in all other cases and I think this makes is a little hard to follow